content/_partials/downloadbanner.html.haml

```diff
@@ -37,7 +37,7 @@
   #ji-download
     %button#download-banner-btn.btn-lg.btn.btn-primary{'data-toggle'=>"popover", 'aria-haspopup'=>"true", 'aria-expanded'=>"false"}
       Download Jenkins
-    = partial('downloadlist.html.haml')
+    = partial('downloadlistcopy.html.haml')
```

Comment:
I don't understand why this file is created and introduced, what purpose does a redundant version of the partial have?